src/components/navigation/DrawerLinkItem.tsx

```diff
@@ -0,0 +1,49 @@
+import { SupportedLanguageTypes } from '@/configs'
+import { hulen_black, hulen_yellow, hulen_yellow_text } from '@/styles'
+import { SanityNavElement } from '@/types/sanity'
+import { Typography } from '@mui/material'
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
+import { MouseEventHandler } from 'react'
+
+/** Drawer link item used in DropDown and in MenuDrawer on mobile
+ */
+export const DrawerLinkItem = ({
+  navElement,
+  onClick,
+  language,
+}: {
+  navElement: SanityNavElement
+  onClick?: MouseEventHandler<HTMLAnchorElement>
+  language: SupportedLanguageTypes
+}) => {
+  const currentPath = usePathname()
+
+  return (
+    <Link
```

Comment:
Next link er litt problematisk. Vi ønsker å ha funksjonaliteten til Next link, men ikke stylingen, som bør kontrolleres fra Material UI.

Jeg har en wrapper i FM jeg bruker for å oppnå dette:
```
import Link from 'next/link'
import type { LinkProps as MuiLinkProps } from '@mui/material'
import { Link as MuiLink } from '@mui/material'
import type { PropsWithChildren } from 'react'

export const LinkWrapper = ({ children, ...rest }: PropsWithChildren<MuiLinkProps>) => {
  return (
    <MuiLink component={Link} {...rest} sx={{ all: 'unset' }}>
      {children}
    </MuiLink>
  )
}
```

Litt shennanigans som skjer her, men det vi gjør er å si at vi har en MuiLink som instansierer komponenten NextLink. Dermed arver den styling og funksjonalitet ifra MuiLink, som så arver selve funksjonaliteten ifra next.

Om du trenger å style den mer spesifikt så kanskje SX på muiLink eller Styled Component på LinKWrapper